tests/integration/tests/enhancements/with-delegate/enhanced-client.test.ts

```diff
@@ -1196,6 +1196,28 @@ describe('Polymorphism Test', () => {
         );
     });
 
+    it('handles very long property names', async () => {
```

Comment:
Hey @irvinzz , thanks for making this PR!

I tried this test case and found it actually passes both before and after this fix. Looking at the code, I believe the problem actually happens when the concrete model's name is very long (instead of a field). Is it what you're running into?